File: src/com/laamella/intellij6502/asm/psi/AssemblyElementType.java
package com.laamella.intellij6502.asm.psi;

import com.intellij.psi.tree.IElementType;
import com.laamella.intellij6502.asm.AssemblyLanguage;
import org.jetbrains.annotations.*;

public class AssemblyElementType extends IElementType {
    public AssemblyElementType(@NotNull @NonNls String debugName) {
        super(debugName, AssemblyLanguage.INSTANCE);
    }
}

File: src/main/java/com/laamella/intellij6502/asm/psi/AssemblyElementType.kt
package com.laamella.intellij6502.asm.psi

import com.intellij.psi.tree.IElementType
import com.laamella.intellij6502.asm.AssemblyLanguage
import org.jetbrains.annotations.*

class AssemblyElementType(@NonNls debugName: String) : IElementType(debugName, AssemblyLanguage.INSTANCE)